deploy/examples/import-external-cluster.sh

```diff
@@ -167,6 +168,20 @@ function importConfigMap() {
   fi
 }
 
+function createInputCommadConfigMap() {
+  if ! $KUBECTL -n "$NAMESPACE" get configmap "$EXTERNAL_COMMAND_CONFIGMAP_NAME" &>/dev/null; then
+    $KUBECTL -n "$NAMESPACE" \
+      create \
+      configmap \
+      "$EXTERNAL_COMMAND_CONFIGMAP_NAME" \
+      --from-literal=command="$EXTERNAL_CLUSTER_USER_COMMAND"
+  else
+    echo "configmap $EXTERNAL_COMMAND_CONFIGMAP_NAME already exists, update it"
+    $KUBECTL -n "$NAMESPACE" patch configmap "$EXTERNAL_COMMAND_CONFIGMAP_NAME" \
+      -p "{'data':{'command':$EXTERNAL_COMMAND_CONFIGMAP_NAME}}"
```

Comment:
Let's try to be consistent with formatting. The `$KUBECTL create X Y --vals` commands elsewhere all use the format below. Let's do the same for this `patch` command to keep consistency.

```
$KUBECTL -n "$NAMESPACE" \
  create \
  X \
  Y \
  --vals
```

Reply:
Done